<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_class</class>
 <widget class="QDialog" name="change_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>240</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="class_NO_label">
        <property name="text">
         <string>class_NO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="class_NO_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="grade_label">
        <property name="text">
         <string>grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="grade_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="person_number_label">
        <property name="text">
         <string>person_number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="person_numberlineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="college_name_label">
        <property name="text">
         <string>college_name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="college_name_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="counselor_label">
        <property name="text">
         <string>counselor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="counselor_lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>class_NO_label</zorder>
  <zorder>class_NO_lineEdit</zorder>
  <zorder>grade_label</zorder>
  <zorder>person_number_label</zorder>
  <zorder>college_name_label</zorder>
  <zorder>counselor_label</zorder>
  <zorder>counselor_lineEdit</zorder>
  <zorder>college_name_lineEdit</zorder>
  <zorder>grade_lineEdit</zorder>
  <zorder>person_numberlineEdit</zorder>
  <zorder>college_name_label</zorder>
  <zorder>counselor_label</zorder>
  <zorder>person_number_label</zorder>
  <zorder>grade_label</zorder>
  <zorder>class_NO_label</zorder>
  <zorder>class_NO_lineEdit</zorder>
  <zorder>counselor_lineEdit</zorder>
  <zorder>college_name_lineEdit</zorder>
  <zorder>grade_lineEdit</zorder>
  <zorder>person_numberlineEdit</zorder>
  <zorder>grade_lineEdit</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>change_class</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>change_class</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
